src/lib/getModelData.ts

```diff
@@ -24,13 +24,13 @@ type Options = {
   width: number
   height: number
   mappings?: Record<string, string>
-  thumbnailType: '3d' | '2d'
+  thumbnailType: 'default' | 'top' | 'front'
```

Comment:
You think it might be a good idea to add this to an enum and name it? It's easier to type everywhere and to edit (if necessary in the future)

Reply:
done 👍